src/Our.Umbraco.BackofficeThemes/packages.config

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="AutoMapper" version="3.3.1" targetFramework="net461" />
+  <package id="ClientDependency" version="1.9.6" targetFramework="net461" />
+  <package id="ClientDependency-Mvc5" version="1.8.0.0" targetFramework="net461" />
+  <package id="Examine" version="0.1.89" targetFramework="net461" />
+  <package id="HtmlAgilityPack" version="1.4.9.5" targetFramework="net461" />
+  <package id="ImageProcessor" version="2.5.6" targetFramework="net461" />
+  <package id="ImageProcessor.Web" version="4.8.7" targetFramework="net461" />
+  <package id="log4net" version="2.0.8" targetFramework="net461" />
+  <package id="Log4Net.Async" version="2.0.4" targetFramework="net461" />
+  <package id="Lucene.Net" version="2.9.4.1" targetFramework="net461" />
+  <package id="Markdown" version="1.14.7" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Identity.Core" version="2.2.1" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Identity.Owin" version="2.2.1" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Mvc" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.SignalR.Core" version="2.2.1" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebApi.WebHost" version="5.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net461" />
+  <package id="Microsoft.IO.RecyclableMemoryStream" version="1.2.2" targetFramework="net461" />
+  <package id="Microsoft.Owin" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.Host.SystemWeb" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.Security" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.Security.Cookies" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Owin.Security.OAuth" version="3.1.0" targetFramework="net461" />
+  <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net461" />
+  <package id="MiniProfiler" version="2.1.0" targetFramework="net461" />
+  <package id="MySql.Data" version="6.9.12" targetFramework="net461" />
+  <package id="Newtonsoft.Json" version="10.0.2" targetFramework="net461" />
+  <package id="Owin" version="1.0" targetFramework="net461" />
+  <package id="semver" version="1.1.2" targetFramework="net461" />
+  <package id="SharpZipLib" version="0.86.0" targetFramework="net461" />
+  <package id="System.Threading.Tasks.Dataflow" version="4.7.0" targetFramework="net461" />
+  <package id="UmbracoCms.Core" version="7.8.0" targetFramework="net461" />
```

Comment:
All those dependencies are coming from installation of UmbracoCms.Core?

Reply:
I cleand a bit up. But these packages ares used to build the solution (normale Umbraco.Core stuff)